client/src/app.js

```diff
@@ -241,7 +256,7 @@ export default function main({ DOM, HTTP, route, storage, scanner: scan$, search
                      , mempool$, mempoolRecent$, feeEst$
                      , tx$, txAnalysis$, openTx$
                      , goAddr$, addr$, addrTxs$, addrQR$
-                     , assetMap$, goAsset$, asset$, assetTxs$
+                     , assetMap$, assetList$, goAssetList$, totalAssets$, goAsset$, asset$, assetTxs$
```

Comment:
`goAssetList$` is an intent and does not belong in the state.

Reply:
I added it to state so that I can access the following params ( startIndex, sortDir, sortField, limit) 

I think it was easier that way...
const { startIndex, sortDir, sortField, limit } = goAssetList